src/shared/components/linkedIconCard/linkedIconCard.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './linkedIconCard.css';
+
+// Stateless component that wraps a link around an icon / img and title
+// Image or Icon are passed as a child to allow more flexibility
+// Props = url, title, and at least one child
+const linkedIconCard = props => (
```

Comment:
All components are required to have the first letter be capitalized: https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized

You're not getting any issues because you've rendered it in JSX with capital letters, but it's good practice to also list the components as such. You could end up with functions inside of an SFC and it helps differentiate functions from the component.

It's annoying that React even let's you do this without an error/warning, imo.